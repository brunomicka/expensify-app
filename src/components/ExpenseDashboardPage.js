import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <br />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);


export default ExpenseDashboardPage;