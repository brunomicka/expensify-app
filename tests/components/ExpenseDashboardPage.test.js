import React from 'react';
import ExpenseDashboardPage  from '../../src/components/ExpenseDashboardPage';
import { shallow } from 'enzyme';

test('should render ExpenseDashboardPage correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
})