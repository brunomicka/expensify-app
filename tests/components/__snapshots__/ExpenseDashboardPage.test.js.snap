// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <br />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
